src/main/java/org/amalitech/ContactBook.java

```diff
@@ -0,0 +1,54 @@
+package org.amalitech;
+
+import java.util.ArrayList;
+
+public class ContactBook {
+ private String name;
+ private String email;
+ ArrayList<Contact> contacts;
```

Comment:
`Array<Contact> contacts = new ArrayList<Contact>()`

But I'd suggest using a set to hold the Contacts, as you'd not want repetitions in the contacts
eg: `Set<Contact> contacts = new HashSet<Contact>()`